docs/api/symbol-package-publish-resource.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Push Symbol Packages, NuGet API | Microsoft Docs
+author:
+- cristinamanu
+- kraigb
+ms.author:
+- cristinamanu
+- kraigb
+manager: skofman
+ms.date: 10/30/2018
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+description: The publish service allows clients to publish new symbol packages.
+keywords: NuGet API push symbol package
+ms.reviewer: karann
+---
+
+# Push Symbol Packages
+
+It is possible to push symbols packages ([snupkg](../create-packages/Symbol-Packages-V2.md)) using the NuGet V3 API.
+These operations are based off of the `SymbolPackagePublish` resource found in the [service index](service-index.md).
+
+## Versioning
+
+The following `@type` value is used:
+
+@type value          | Notes
+-------------------- | -----
+SymbolPackagePublish/4.9.0 | The initial release
+
+## Base URL
+
+The base URL for the following APIs is the value of the `@id` property of the `SymbolPackagePublish/4.9.0` resource in the
+package source's [service index](service-index.md). For the documentation below, nuget.org's URL is used. Consider 
+`https://www.nuget.org/api/v2/symbolpackage` as a placeholder for the `@id` value found in the service index.
+
+
+## HTTP methods
+
+The `PUT` HTTP method is supported by this resource. 
+
+## Push a symbol package
+
+nuget.org supports pushing new symbol packages format ([snupkg](../create-packages/Symbol-Packages-V2.md)) using the following API. 
+
+    PUT https://www.nuget.org/api/v2/symbolpackage
+
+Symbol packages with the same ID and version
+can be submitted multiple times. A symbol package will be rejected in the following cases.
+- A package with the same ID and version does not exist.
+- A symbol package with the same ID and version was pushed but is not yet published.
+- The symbol package ([snupkg](../create-packages/Symbol-Packages-V2.md)) is invalid (see [symbol package constraints](../create-packages/Symbol-Packages-V2.md)).
+
+For snupkg constraints see [symbol package constraints](../create-packages/Symbol-Packages-V2.md).
```

Comment:
You mention the "snupkg constraints" twice in close succession. Is this necessary?

Reply:
I removed the last one to avoid confusion/duplication of information.